src/epd_driver/include/epd_driver.h

```diff
@@ -13,6 +13,14 @@ extern "C" {
 #include <stdbool.h>
 #include <stdint.h>
 
+/// Display software rotation. Recalculates x, y of draw_pixel, fonts and partial update area
```

Comment:
Maybe elaborate on the documentation, like mentioning epd_set_rotation for setting the rotation and that it sets the rotation for the purposes of the drawing and font functions. I think "recalculate" sounds a bit misleading.